doc/whats_new/v1.5.rst

```diff
@@ -550,29 +550,29 @@ Changelog
 Thanks to everyone who has contributed to the maintenance and improvement of
 the project since version 1.4, including:
 
-101AlexMartin, Abdulaziz Aloqeely, Adam J. Stewart, Adam Li, Adarsh Wase, Adrin 
-Jalali, Advik Sinha, Akash Srivastava, Akihiro Kuno, Alan Guedes, Alexis 
-IMBERT, Ana Paula Gomes, Anderson Nelson, Andrei Dzis, Arnaud Capitaine, Arturo 
-Amor, Aswathavicky, Bharat Raghunathan, Brendan Lu, Bruno, Cemlyn, Christian 
-Lorentzen, Christian Veenhuis, Cindy Liang, Claudio Salvatore Arcidiacono, 
-Connor Boyle, Conrad Stevens, crispinlogan, davidleon123, DerWeh, Dipan Banik, 
-Duarte São José, DUONG, Eddie Bergman, Edoardo Abati, Egehan Gunduz, Emad 
-Izadifar, Erich Schubert, Filip Karlo Došilović, Franck Charras, Gael 
-Varoquaux, Gönül Aycı, Guillaume Lemaitre, Gyeongjae Choi, Harmanan Kohli, 
-Hong Xiang Yue, Ian Faust, itsaphel, Ivan Wiryadi, Jack Bowyer, Javier Marin 
-Tur, Jérémie du Boisberranger, Jérôme Dockès, Jiawei Zhang, Joel Nothman, 
-Johanna Bayer, John Cant, John Hopfensperger, jpcars, jpienaar-tuks, Julian 
-Libiseller-Egger, Julien Jerphanion, KanchiMoe, Kaushik Amar Das, keyber, 
-Koustav Ghosh, kraktus, Krsto Proroković, ldwy4, LeoGrin, lihaitao, Linus 
-Sommer, Loic Esteve, Lucy Liu, Lukas Geiger, manasimj, Manuel Labbé, Manuel 
-Morales, Marco Edward Gorelli, Maren Westermann, Marija Vlajic, Mark Elliot, 
-Mateusz Sokół, Mavs, Michael Higgins, Michael Mayer, miguelcsilva, Miki 
-Watanabe, Mohammed Hamdy, myenugula, Nathan Goldbaum, Naziya Mahimkar, Neto, 
-Olivier Grisel, Omar Salman, Patrick Wang, Pierre de Fréminville, Priyash 
-Shah, Puneeth K, Rahil Parikh, raisadz, Raj Pulapakura, Ralf Gommers, Ralph 
-Urlus, Randolf Scholz, Reshama Shaikh, Richard Barnes, Rodrigo Romero, Saad 
-Mahmood, Salim Dohri, Sandip Dutta, SarahRemus, scikit-learn-bot, Shaharyar 
-Choudhry, Shubham, sperret6, Stefanie Senger, Suha Siddiqui, Thanh Lam DANG, 
-thebabush, Thomas J. Fan, Thomas Lazarus, Thomas Li, Tialo, Tim Head, Tuhin 
-Sharma, VarunChaduvula, Vineet Joshi, virchan, Waël Boukhobza, Weyb, Will 
+101AlexMartin, Abdulaziz Aloqeely, Adam J. Stewart, Adam Li, Adarsh Wase, Adrin
```

Comment:
it's pre-commit how did it

Reply:
Fixed it by checkouting the file from `origin/main` and merging main into this branch.